packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -143,7 +143,8 @@ export const messages = {
   force_lightning_event_create_text: 'SFDX: Create Aura Event',
   force_lightning_interface_create_text: 'SFDX: Create Aura Interface',
   force_function_create_text: 'SFDX: Create Function',
-  force_function_start_text: 'SFDX: Start Function',
+  force_function_container_start_text: 'SFDX: Start Container Function',
+  force_function_containerless_start_text: 'SFDX: Start Local Function',
```

Comment:
DO I need to do anything with the package.nls.ja.json if new values are added here?

Reply:
Not required. We rely on an external contributor (and until then, it defaults to English).